src/kit.ts

```diff
@@ -223,7 +223,7 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): Promise
 
           if (isMake && isMingwTool) {
             gccKit.preferredGenerator = {name: 'MinGW Makefiles'};
-            gccKit.environmentVariables = {PATH: ENV_PATH};
+            gccKit.environmentVariables = {CMT_MINGW_PATH: ENV_PATH};
```

Comment:
Do we need to force a rescan of kits?  (i.e. will developers get this fix without a rescan?)

Reply:
If we can force a rescan the first time the extension is activated after updating to this version of CMT, great and yes!  But I don't want to otherwise force scans...though a scan will be necessary to fix the problem.